samples/SimpleServiceSample/SimpleServiceSample.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Worker">
-  
+
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+  	<TargetFramework>net6.0</TargetFramework>
```

Comment:
Tabs 😞

Reply:
I am on it, I will push a commit for it.

Reply:
Done.
<img width="731" alt="image" src="https://user-images.githubusercontent.com/68565441/146656631-d7de9220-e681-42f0-b27f-267a8aa6be9b.png">